readthedocs/core/views/serve.py

```diff
@@ -152,23 +152,24 @@ def serve_docs(
     """Exists to map existing proj, lang, version, filename views to the file format."""
     if not version_slug:
         version_slug = project.get_default_version()
+
     try:
-        version = project.versions.public(request.user).get(slug=version_slug)
+        version = project.versions.get(slug=version_slug)
```

Comment:
This change needs to be looked at carefully since `.public` has a different meaning than `.get` under our corporate site.

I'm not saying that it's wrong, but just making notes about it when proper review.